variables.js

```diff
@@ -15,6 +15,14 @@
     2. Use console.log() to output the value of each variable.
  ********************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let personName = "ali";
+
+console.log(personName);
+let age = "15";
+console.log(age);
+let isHappy = "true";
```

Comment:
Same thing for the boolean value, you should assign it directly without enclosing it in double quotation marks. Here's the corrected version of the code:
**let isHappy = true;**